components/ca-crud.html

```diff
@@ -729,6 +743,20 @@
                         self.onaction(action, data, dlg);
                         return false;
                     }
+
+                    // find the navigation for the button that was clicked
+                    const nav = labelToNav[button];
+
+                    if (nav) {
+                        // we want to raise a navigation action event
+                        const event = new CustomEvent('ca-crud-action-nav', {
+                            detail: data,
+                            bubbles: true
+                        });
+                        self.dispatchEvent(event);
+                        return true;
```

Comment:
I believe it is required to ensure the dialog that is currently open is closed. So yes, I think it is required